packages/babel-plugin-proposal-pattern-matching/package.json

```diff
@@ -12,7 +12,8 @@
     "@babel/plugin-syntax-pattern-matching": "7.0.0-beta.32"
   },
   "peerDependencies": {
-    "@babel/core": "7.0.0-beta.32"
+    "@babel/core": "7.0.0-beta.32",
+    "@babel/template": "7.0.0-beta.32"
```

Comment:
sorry you can also just use `@babel/core` just it contains template for you!

`import { template, types as t } from "@babel/core";`

Reply:
@hzoo OK